analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -261,6 +280,36 @@ private static void VerifyBuildErrors(ImmutableArray<Diagnostic> diagnostics, Co
 
         private static string IssueType(bool isPrimary) => isPrimary ? "primary" : "secondary";
 
+        internal class FileNameSourceText
+        {
+            public string FileName { get; set; }
+            public SourceText SourceText { get; set; }
+            public FileNameSourceText(string fileName, SourceText sourceText)
+            {
+                FileName = fileName;
+                SourceText = sourceText;
+            }
+
+            public FileNameIssueLocations ToIssueLocations() =>
```

Comment:
I'd prefer to remove this method. It's intended to be used only inside this class and the other usage is a very special case.

Reply:
It is pretty long one. Also, since I've hidden all the members - nothing is seen outside. Let's keep it as it is, since it's really a space-saver